autotest/gdrivers/tiledb_write.py

```diff
@@ -78,38 +88,50 @@ def test_tiledb_write_update():
 
     gdaltest.tiledb_drv = gdal.GetDriverByName('TileDB')
 
-    new_ds = gdaltest.tiledb_drv.Create('tmp/tiledb_update', 20, 20, 1, gdal.GDT_Byte)
-    new_ds.GetRasterBand(1).WriteArray(np.zeros((20, 20)))
-    del new_ds
+    for mode in [ 'BAND', 'PIXEL' ]:
```

Comment:
a more pytest'ic way would be to use pytest.parametrize (see autotest/gdrivers/fits.py for an example)